tests/main/snap-user-service-socket-activation/task.yaml

```diff
@@ -40,7 +40,7 @@ execute: |
     # small hole to allow that.
     chmod +x /run/user/"$(id -u test)" ~test
     # The runtime directory will be removed by now.
-    tests.cleanup defer chmod -x ~test
+    tests.cleanup defer chmod o-x ~test
```

Comment:
can you `stat -c '%a' ~test` before and restore the same in defer?

Reply:
Done